dashboard/engines/hoc_legacy/lib/hoc_legacy/tutorials.rb

```diff
@@ -30,7 +32,17 @@ def refresh
       private def fetch_all
         CdoContentful::CsForAll::Entry::Tutorial.
           find_each(order: FETCH_ORDER, limit: FETCH_LIMIT).
```

Comment:
How about we search for tutorials that have a tutorial id instead? Lacking a tutorial id should simply mean it's not a valid tutorial.